report/11_Systems-Design.md

```diff
@@ -1 +1,14 @@
-Design of your ITU-MiniTwit systems
+### System Design 
+The MiniTwit application follows the Repository design pattern, with a database abstraction layer, a backend with repositories and a frontend that receives data from these repositories. 
```

Comment:
Is it not only part of the application that uses the design pattern. Not the entire application that follows it?
I would not mention the design pattern here.